forms/publishdialog.ui

```diff
@@ -90,11 +90,34 @@
      </property>
     </item>
    </widget>
+   <widget class="QComboBox" name="comboBox_representation">
+    <property name="geometry">
+     <rect>
+      <x>180</x>
+      <y>80</y>
```

Comment:
If the previous comment is accepted, this value would be 78 to keep the coherence of 34 pixels.